### PR TITLE
Added snackbar instead of toast and format code

### LINKED ISSUE #12552: Show a snackbar instead of a toast
There is a list of actoins which show a toast but a snackbar should be shown 

Card browser -> select a card ->
- Reposition card
- Reschedule card
- Reset progress
- Change deck -> Create a new deck -> Created deck

Deck picker -> long tap on a deck ->
- Rename deck
- Create subdeck

Settings 
- Sync -> Force full sync -> OK
- About -> Copy debug info

_Originally posted by @criticalAY in https://github.com/ankidroid/Anki-Android/issues/12487#issuecomment-1255396087_

-----
AnkiDroid Version = 2.16alpha85
New schema = true
Scheduler = std3
DatabaseV2 Enabled = true

### PR DESCRIPTION
**Pull request by pratyaksh1610**
## Purpose / Description
Added snackbar instead of toast and format code

## Fixes
Fixes #12552 

## How Has This Been Tested?

Tested on emulator Ktlint checks passed

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt [hunk 1]
```diff
@@ -105,9 +107,15 @@ class AboutFragment : Fragment() {
         val clipboardManager = requireActivity().getSystemService(CLIPBOARD_SERVICE) as ClipboardManager?
         if (clipboardManager != null) {
             clipboardManager.setPrimaryClip(ClipData.newPlainText("$appName v$pkgVersionName", debugInfo))
-            UIUtils.showThemedToast(context, getString(R.string.about_ankidroid_successfully_copied_debug), true)
+            showSnackbar(
+                getString(R.string.about_ankidroid_successfully_copied_debug),
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt:111]:
```suggestion
                R.string.about_ankidroid_successfully_copied_debug,
```

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt [hunk 2]
```diff
@@ -105,9 +107,15 @@ class AboutFragment : Fragment() {
...
+                Snackbar.LENGTH_SHORT
+            )
         } else {
-            UIUtils.showThemedToast(context, getString(R.string.about_ankidroid_error_copy_debug_info), false)
+            showSnackbar(
+                getString(R.string.about_ankidroid_error_copy_debug_info),
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt:116]:
```suggestion
                R.string.about_ankidroid_error_copy_debug_info,
```

**Review by BrayanDSO — COMMENTED** [review]
Only nits

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt (hunk 1 above)
Comment (pratyaksh1610) [reply AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt]: Done

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt (hunk 2 above)
Comment (pratyaksh1610) [reply AnkiDroid/src/main/java/com/ichi2/anki/preferences/AboutFragment.kt]: Done

**Review by BrayanDSO — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**